### DISCUSSION TITLE
[Feature] Grandma Mode

### DISCUSSION THREAD

**Discussion by Frankenberrypi** [immich-app/immich#9866] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add the ability to open the app directly to the Sharing section or a specific shared album.  Telling someone "tap the rainbow flower icon to see grand kids pictures" is much easier that trying to explain sharing to them.  

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by matticas** [comment]
Can I possibly add;
Also direct it to open directly to an identified album.
In my case; open directly to the "Offspring" album - the grandkids.

**Comment by dkerlee** [comment]
I thought I'd come up with this exact idea on my own! But yes: having this feature was one of the cool things about g-photos, it keeps the grandparents in the loop with minimal effort from parents. Huge boon to familial sharing, and family Immich viewing!

**Comment by matticas** [comment]
I've made Grandparent mode by creating a new user, sharing an album with
them, not alloying them to modify it, then setting the album URL as their
browser homepage.

Works.  Grandparents see daily photos of the kids without us doing anything
really, besides adding photos of the kids to that album.

This is where auto people detection -straight to album would be useful.

--Matt
